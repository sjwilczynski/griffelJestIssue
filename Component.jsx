import * as React from "react";
import { makeStyles } from "@fluentui/react-components";

const useStyles = makeStyles({
  root: {
    containerType: "inline-size",
  },
  element: {
    backgroundColor: "blue",
    "@container (width > 600px)": {
      backgroundColor: "red",
    },
  },
});

export const Component = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <div className={classes.element}>Hello World</div>
    </div>
  );
};
