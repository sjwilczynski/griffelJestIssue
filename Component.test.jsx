/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react";
import { Component } from "./Component";

describe("Component", () => {
  it("renders", () => {
    const { container } = render(<Component />);
    expect(container).toMatchSnapshot();
  });
});
