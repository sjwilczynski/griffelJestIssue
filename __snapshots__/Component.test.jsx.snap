// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<div>
  <div
    class="___gnf9xs0_0000000 f138vucn"
  >
    <div
      class="___799s900_0000000 f1bh81bl f1wd2ja7"
    >
      Hello World
    </div>
  </div>
</div>
`;
